README.md

```diff
@@ -23,16 +25,17 @@ These steps come from the [extension dev page](https://developer.mozilla.org/en-
     - `-purgecaches` stops FF from caching files, so you can instantly (?) see changes
 
 ### Setup specific to this addon
-1. Set up https on your local machine.
-  - Warning: this sucks, but is necessary.
-  - Here's a gist with how I got this working (yosemite / built-in apache): https://gist.github.com/6a68/40b5eda14c82a25e253b
-1. Configure the addon to use your local copy of the [iframe code](https://github.com/mozilla/universal-search-content):
+1. If you're hacking on the [iframe](https://github.com/mozilla/universal-search-content), then configure your profile to use a local copy:
   - Set these two prefs in about:config:
   - `services.universalSearch.frameURL`: the complete iframe URL
     - default is 'https://d1fnkpeapwua2i.cloudfront.net/index.html'
   - `services.universalSearch.baseURL`: the iframe's base URL
```

Comment:
Unrelated, but I feel like lines 30-34 should be indented another level since they seem like a sub-list of L29.